api/client_test.go

```diff
@@ -1052,3 +1081,45 @@ func TestClient_SetReadYourWrites(t *testing.T) {
 		})
 	}
 }
+
+func TestClient_SetCloneToken(t *testing.T) {
+	tests := []struct {
+		name  string
+		calls []bool
+	}{
+		{
+			name:  "false",
+			calls: []bool{false},
+		},
+		{
+			name:  "true",
+			calls: []bool{true},
+		},
+		{
+			name:  "multi",
+			calls: []bool{true, false, true},
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			c := &Client{
+				config: &Config{},
+			}
+
+			var expected bool
+			for _, v := range tt.calls {
+				actual := c.CloneToken()
+				if expected != c.CloneToken() {
```

Comment:
nit: should we compare to `actual` here since we already have the value captured?

Reply:
Good call! Fixed in https://github.com/hashicorp/vault/pull/13515/commits/816fc3083787a6e6df795a0689d23b11177eda11